services/ansible/ansible-quality.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const { floorCount } = require('../../lib/color-formatters')
+
+const ansibleContentSchema = Joi.object({
+  quality_score: Joi.number().required(),
```

Comment:
It seems like some roles have a `quality_score` value of `null` e.g: https://galaxy.ansible.com/api/v1/content/24559/
At the moment we'll error with "invalid response data" in that case.. do we know what a `null` score indicates? If so, would it be better to allow null here and then throw an exception to provide a friendlier error message in that case? I'm genuinely undecided on this.. maybe "invalid response data" is fine?

Reply:
Seems like `No Score Available` is the sensible error here. Lets accept `null` in the schema. If its `null`, throw an `InvalidResponse` with that message https://github.com/badges/shields/blob/5233a25ddcee9cc14eb8b67329aa019af613325e/services/errors.js#L40